src/utils/feedback.ts

```diff
@@ -52,6 +52,17 @@ export function getCreatePricingExchangeFeedback(
   }
 }
 
+export function getCreateFreePricingFeedback(
+  dtSymbol: string
+): { [key: number]: string } {
+  return {
+    99: 'Creating dispenser...',
+    0: `Proposing ${dtSymbol} minter...`,
+    1: 'Dispenser approving minter...',
```

Comment:
I know this is what is technically happening in background but nobody knows what this means. Either we explain what is happening in the UI text in addition to those steps, or adapt those step messages

Reply:
How about this below, is it better? 
`99: 'Creating ${dtSymbol} faucet...'`
`0: 'Setting faucet as minter ...'`
`1: 'Approving minter...'  `